src/css/LeetcodeText.module.css

```diff
@@ -0,0 +1,54 @@
+.container {
+  margin-left: 14%;
+  margin-right: 18%;
+  padding-top: 73px;
+  padding-bottom: 45%;
+}
+
+.title {
+  margin: 0;
+  margin-bottom: 16px;
+  padding-left: 1%;
```

Comment:
Don't think you need padding-left: 1% and can remove margin-bottom if you conditionally render through the js component.